File: AreaTriangulo.scala
import scala.io.StdIn.readLine
object AreaTSC {
  def main(args: Array[String]) = {
  //Declaracion de variables
  println("Ejercicio 01: Area Triangulo")
  //Datos de entrada
  println("Ingrese base:")
  val b = readLine().toInt
  println("Ingrese altura:")
  val h = readLine().toInt
  //Proceso
  val area=(b*h)/2
  //Datos de salida 
  println("El are del Triangulo es:"+area)
  }
}

File: AreaTriangulo.cpp
#include <iostream>
int main () {
  //definir variables
  printf("Ejercicio 01: Area de un triangulo\n");
  int b=0, h=0, area=0;
  // Datos de entrada
  printf("Ingrese base:");
  scanf("%i", &b);
  printf("INgrese altura:");
  scanf("%i", &h);
  // proceso
  area=(b*h)/2;
  // Datos de salida
  printf("Area de un Triangulo es:%i %s", area, "\n");

  return 0;
}
